src/views/Timeline.vue

```diff
@@ -271,6 +282,24 @@ export default {
     })
   },
   methods: {
+    setConnectionStatusNotification () {
+      switch (this.peerConnectionStatus) {
+        case 'PEER_CONNECTING':
+          this.connectionBarText = 'UI App is connecting to Ambianic Edge Device'
```

Comment:
@vickywane small typo: 'UI App is connecting to an Ambianic Edge Device'.

Also for small screens, I wonder if this is too long of a label. May something shorter like:
"Connecting to Ambianic Edge device" would work better without losing quality of the information.

Reply:
Okay, this will fit in nicely